### PR TITLE
Testing To Do list: part 1

### PR DESCRIPTION
**Pull request by khanh-devos**
In this part1 activity, we will use Jest to test functions:
- addNew() and deleteTasks()
- build a mockData to replace the localStorage.

### CODE REVIEW

File: test/part1.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+  if (newTask.description.length > 0) {
+    const check = data.getData();
+    return check[check.length - 1];
+  }
+  throw new Error('task cannot be an empty string');
+};
+
+export const deleteTasks = (checks) => {
+  // const checks = document.querySelectorAll('input[type="checkbox"]');
```
Comment (TedLivist) [inline test/part1.js]: - [x] Let's endeavour to remove commented code so as to avoid confusion in reading the code.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @khanh-devos, @H-Ramalan and @ClarenceAtim,

Good job so far!
✔️ Well-written and passing tests
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test/part1.js (hunk 1 above)
Comment (khanh-devos) [reply test/part1.js]: Hello @TedLivist ,
Thanks for your your suggestion.

**Review by topeogunleye — APPROVED** [review]
### Hi @khanh-devos, @H-Ramalan, @ClarenceAtim,

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Removed commented-out code ✔️ 

## [Optional] suggestions:
* For future pull requests, please notice that your branch name `part1` is not descriptive enough of the changes in this branch and does not follow the convention of being named after a feature or action. For example, you could name your branch `added-unit-test.` This makes it easier to track the changes and the purpose of each branch..
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by khanh-devos** [comment]
Hello @topeogunleye,
Thank you for your remind.

**Pull request merged**